configuration/pih/scripts/global/pihServices.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Adds javascript functions to the pihemr
+ * Must be instantiated with jQuery instance
+ */
+class PihServices {
```

Comment:
Compared to the above, I don't think we have anything in HTML Form Entry that provides utility methods for interacting with OpenMRS RESTfully, so it would likely make sense to add these to HFE directly, unless there is something specific here to PIH I missed?

Reply:
Well, there is also nothing specific to HFE about wanting to access REST services.  So why bundle something with HFE that could have broader uses?  I could see the REST module providing javascript library resources - that would make a lot of sense.  But that's outside the scope of what I'm doing here.